docs/hub/webhooks-guide-auto-retrain.md

```diff
@@ -0,0 +1,176 @@
+# Webhook guide: Setup an automatic system to re-train a model when a dataset changes
+
+This guide will help walk you through the set up of an automatic training pipeline on the HuggingFace platform
+using HF Webhooks, Spaces and AutoTrain.
+
+We will build a Webhook that listens to changes on an image classification dataset and triggers a fine-tuning
+of [microsoft/resnet-50](https://huggingface.co/microsoft/resnet-50) using [AutoTrain](https://huggingface.co/autotrain).
+
+
+
+## Prerequisite: Upload your dataset to the Hub
+
+We will use a [simple image classification dataset](https://huggingface.co/datasets/huggingface-projects/auto-retrain-input-dataset) for the sake
+of the example. Learn more about uploading your data to the Hub here: https://huggingface.co/docs/datasets/upload_dataset.
+
+![dataset](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/webhooks-guides/002-auto-retrain/dataset.png)
+
+## Create a webhook to react to the dataset's changes
+
+First, let's create a webhook from your settings: https://huggingface.co/settings/webhooks.
+
+- Select your dataset as the target repository. We will target [huggingface-projects/input-dataset](https://huggingface.co/datasets/huggingface-projects/input-dataset) in this example.
+- You can put a dummy Webhook URL for now. Defining your webhook will let you take a look at the events that are going to be sent to it. You can also replay them, which will be useful for debugging!
+- Input a secret to make it more secure.
+- Subscribe to "Repo update" events as we want to react to data changes
+
+Your webhook will look like this:
+
+![webhook-creation](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/webhooks-guides/002-auto-retrain/webhook-creation.png)
```

Comment:
very small nit but it might be better to take screenshot on a mac os machine for consistency with rest of the doc, IF you have one :) (no big deal otherwise...)

Reply:
boooooooooooooooo